src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,57 @@
 package core.basesyntax.impl;
 
+import core.basesyntax.Box;
 import core.basesyntax.Storage;
+import java.util.ArrayList;
+import java.util.List;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+
+    private List<Box> storage = new ArrayList<>();
+
+    public int getIndex(K key) {
```

Comment:
```suggestion
    private int getIndex(K key) {
```

we can make this method private and place it after public as helper one